src/doc/rustc/src/platform-support.md

```diff
@@ -385,5 +385,8 @@ target | std | host | notes
 [`x86_64h-apple-darwin`](platform-support/x86_64h-apple-darwin.md) | ✓ | ✓ | macOS with late-gen Intel (at least Haswell)
 [`x86_64-unknown-linux-none`](platform-support/x86_64-unknown-linux-none.md) | * |  | 64-bit Linux with no libc
 `aarch64-unknown-ferrocenecoretest` | ✓ | ✓ | Internal target for tests
```

Comment:
would keep this placed at the end, to separate it from the rest since it's a special target

Reply:
That will keep causing issues with new targets being added again though, so technically it is a trade off. Maybe adding our targets to the top would be better

Reply:
that targets are sorted, so we could choose either top or bottom